File: TheHuxley/Python/691.py
def greater(num1, num2):
    if num1 < num2:
        print("{0} {1}".format(num1, num2))
    else:
        print("{0} {1}".format(num2, num1))

num1, num2 = input().split()

greater(int(num1), int(num2))

File: TheHuxley/Cpp/691.cpp
#include <iostream>
using namespace std;

int main(){
    int num1, num2;
    cin >> num1 >> num2;
    if(num1 < num2)  cout << num1 << ' ' << num2;
    else cout << num2 << ' ' << num1;

    return 0;
}
